### ISSUE TITLE
Does not live reload on CSS change while using incremental

### ISSUE THREAD

**Issue by hackedunit** [awood/hawkins#11]
I'm seeing an issue where it does not live reload, on CSS changes (SCSS specifically) while passing `--incremental` to `jekyll liveserve`. Is this the expected behaviour?

**Comment by awood** [comment]
No, that's not expected.  Would you mind telling me what version of Jekyll you're using?  Also if you wouldn't mind doing a test run with `--verbose` and then modifying your SCSS file and posting the output, that would be helpful.
